requirements/requirements.txt

```diff
@@ -10,3 +10,4 @@ pandas~=1.0
 pydantic~=1.1
 tqdm~=4.23
 ujson~=1.35
+xgboost>=0.90
```

Comment:
I don't think we want to generally depend on xgboost. You can create a new extras requirement for that.

Reply:
Removed.